Libraries/LibWeb/TrustedTypes/TrustedTypePolicy.cpp

```diff
@@ -182,4 +185,56 @@ WebIDL::ExceptionOr<JS::Value> TrustedTypePolicy::get_trusted_type_policy_value(
     return policy_value;
 }
 
+// https://www.w3.org/TR/trusted-types/#process-value-with-a-default-policy-algorithm
+WebIDL::ExceptionOr<JS::Value> process_value_with_a_default_policy(TrustedTypeName trusted_type_name, JS::Object& global, Variant<GC::Root<TrustedHTML>, GC::Root<TrustedScript>, GC::Root<TrustedScriptURL>, Utf16String> input, InjectionSink sink)
+{
+    auto& vm = global.vm();
+    auto& realm = *vm.current_realm();
+
+    // 1. Let defaultPolicy be the value of global’s trusted type policy factory’s default policy.
+    auto const& default_policy = as<HTML::WindowOrWorkerGlobalScopeMixin>(global).trusted_types()->default_policy();
+
+    // This algorithm routes a value to be assigned to an injection sink through a default policy, should one exist.
+    // FIXME: Open an issue upstream. It is not immediately clear what to do if the default policy does not exist.
```

Comment:
Please do open an issue! It looks like https://github.com/w3c/trusted-types/issues/ is the place for that.